kratos/python/add_factories_to_python.cpp

```diff
@@ -62,6 +91,38 @@ void  AddFactoriesToPython(pybind11::module& m)
      .def("Has",&PreconditionerFactoryType::Has)
     ;
 
+    //////////////////////////////////////////////////////////////7
+    //HERE THE TOOLS TO REGISTER CONVERGENCE CRITERIA
+    py::class_<ConvergenceCriteriaFactoryType, ConvergenceCriteriaFactoryType::Pointer >(m, "ConvergenceCriteriaFactory")
```

Comment:
Here instead of doing for each one can't we expose the base factory's `Create` and `Has` just once and the derived classes have it then ?

Reply:
I will wait until there is a decision by the @KratosMultiphysics/technical-committee

Reply:
Please consider doing this !

Reply:
I think you haven't noticed that intense use of templates makes me register X times the same methods, because each factory is deriving of a factory depending of its own type

Reply:
I did a common class for the only method that does not require template, the Has method